### PR TITLE
feat: Add Suggestion To Use Bulk URL Import

### LINKED ISSUE #2928: [BUG] - "Looking for Migrations" Link on create recipe page not working
### First Check

- [X] This is not a feature request.
- [X] I added a very descriptive title to this issue (title field is above this).
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [X] This issue can be replicated on the demo site (https://demo.mealie.io/).

### What is the issue you are experiencing?

The link on the create recipe page is not working

### Steps to Reproduce

1. Go to create recipe page
2. Klick on "Looking for Migration" Link

### Please provide relevant logs

None

### Mealie Version

Version: nightly
Build: 8ca5a9454e139d4c874cbe50182f61e1f701f811

### Deployment

Docker (Linux), Docker (Windows)

### Additional Deployment Details

_No response_

### PR DESCRIPTION
**Pull request by michael-genson**
## What type of PR is this?

_(REQUIRED)_

- feature

## What this PR does / why we need it:

_(REQUIRED)_

One thing that's bugged me is that some of the "create recipe" options are hidden. You basically have to click on "Import Recipe" to get to the create page, _then_ switch to the "hidden" one.

This PR makes it a bit easier for users to stumble upon the bulk URL importer. There are still two other "hidden" menu options (scraper debugger and .zip importer) but the former makes sense (most people won't use it, and those that will use it will find it) and the latter should be addressed along with https://github.com/mealie-recipes/mealie/issues/2802.

I originally considered putting the hidden options in the main sidebar, but I found it too cluttered, I like how simple it is currently.

## Which issue(s) this PR fixes:

_(REQUIRED)_

Fixes https://github.com/mealie-recipes/mealie/issues/2928

## Special notes for your reviewer:

_(fill-in or delete this section)_

If there's a better way to handle the hyperlink let me know. I used `href` because using `router-link` won't update the menu. I considered rigging up an emit so the parent component can handle it, but it seemed like more of a refactor than it was worth.

## Testing

_(fill-in or delete this section)_

Manually

### CODE REVIEW

File: frontend/pages/g/_groupSlug/r/create/url.vue [hunk 1]
```diff
@@ -4,7 +4,8 @@
       <div>
         <v-card-title class="headline"> {{ $t('recipe.scrape-recipe') }} </v-card-title>
         <v-card-text>
-          {{ $t('recipe.scrape-recipe-description') }}
+          <p>{{ $t('recipe.scrape-recipe-description') }}</p>
+          <p>{{ $t('recipe.scrape-recipe-have-a-lot-of-recipes') }} <a :href="bulkImporterTarget">{{ $t('recipe.scrape-recipe-suggest-bulk-importer') }}</a>.</p>
```
Comment (Kuchenpirat) [inline frontend/pages/g/_groupSlug/r/create/url.vue:8]: If you want to use router-link. This is working for me:

```suggestion
          <p>{{ $t('recipe.scrape-recipe-have-a-lot-of-recipes') }} <router-link :to="bulkImporterTarget">{{ $t('recipe.scrape-recipe-suggest-bulk-importer') }}</router-link>.</p>


```

File: frontend/pages/g/_groupSlug/r/create/url.vue [hunk 2]
```diff
@@ -94,6 +95,11 @@ export default defineComponent({
     const router = useRouter();
     const tags = useTagStore();
 
+    const bulkImporterTarget = computed(() => {
+      const urlIndex = route.value.fullPath.lastIndexOf("/url");
+      return route.value.fullPath.slice(0, urlIndex) + "/bulk";
+    });
```
Comment (Kuchenpirat) [inline frontend/pages/g/_groupSlug/r/create/url.vue]: I think you can simplify that and just use the already computed groupSlug:

```suggestion
    const bulkImporterTarget = computed(() => `/g/${groupSlug.value}/r/create/bulk`);
```

Comment (Kuchenpirat) [reply frontend/pages/g/_groupSlug/r/create/url.vue:101]: I see what you mean with won't update the menu, forget this

File: frontend/pages/g/_groupSlug/r/create/url.vue (hunk 1 above)
Comment (Kuchenpirat) [reply frontend/pages/g/_groupSlug/r/create/url.vue:8]: I see what you mean with won't update the menu, forget this

File: frontend/pages/g/_groupSlug/r/create/url.vue (hunk 2 above)
Comment (michael-genson) [reply frontend/pages/g/_groupSlug/r/create/url.vue]: I feel like there was a reason I did it this way, but I can't remember. Thanks!

Comment (Kuchenpirat) [reply frontend/pages/g/_groupSlug/r/create/url.vue]: only reason i can think of is that is uses the route. 
So if somebody would change those (can't imagine why somebody would do that) you would not have to update the component.

**Review by Kuchenpirat — APPROVED** [review]
LGTM

**Pull request merged**